make/modules/java.desktop/lib/Awt2dLibraries.gmk

```diff
@@ -144,25 +139,26 @@ $(eval $(call SetupJdkLibrary, BUILD_LIBAWT, \
     DISABLED_WARNINGS_microsoft := 4244 4996, \
     DISABLED_WARNINGS_microsoft_awt_Toolkit.cpp := 4267, \
     LDFLAGS_macosx := -L$(INSTALL_LIBRARIES_HERE), \
-    LDFLAGS_windows := -delayload:user32.dll -delayload:gdi32.dll \
-        -delayload:shell32.dll -delayload:winmm.dll \
-        -delayload:winspool.drv -delayload:imm32.dll \
-        -delayload:ole32.dll -delayload:comdlg32.dll \
-        -delayload:comctl32.dll -delayload:shlwapi.dll, \
-    LIBS_unix := -ljvm -ljava $(LIBM), \
-    LIBS_linux :=  $(LIBDL), \
+    LDFLAGS_windows := -delayload:comctl32.dll -delayload:comdlg32.dll \
+        -delayload:gdi32.dll -delayload:imm32.dll -delayload:ole32.dll \
+        -delayload:shell32.dll -delayload:shlwapi.dll -delayload:user32.dll \
+        -delayload:winmm.dll -delayload:winspool.drv, \
+    JDK_LIBS_unix := -ljava -ljvm, \
+    JDK_LIBS_macosx := -lmlib_image, \
+    JDK_LIBS_windows := $(WIN_JAVA_LIB) jvm.lib, \
+    LIBS_unix := $(LIBM), \
+    LIBS_linux := $(LIBDL), \
     LIBS_aix := $(LIBDL),\
-    LIBS_macosx := -lmlib_image \
+    LIBS_macosx := \
+        -framework ApplicationServices \
+        -framework AudioToolbox \
         -framework Cocoa \
-        -framework OpenGL \
-        -framework Metal \
         -framework JavaRuntimeSupport \
-        -framework ApplicationServices \
-        -framework AudioToolbox, \
-    LIBS_windows := kernel32.lib user32.lib gdi32.lib winspool.lib \
-        imm32.lib ole32.lib uuid.lib shell32.lib \
-        comdlg32.lib winmm.lib comctl32.lib shlwapi.lib \
-        delayimp.lib jvm.lib $(WIN_JAVA_LIB) advapi32.lib, \
+        -framework Metal \
+        -framework OpenGL, \
+    LIBS_windows := advapi32.lib comctl32.lib comdlg32.lib delayimp.lib \
+        gdi32.lib imm32.lib kernel32.lib ole32.lib shell32.lib shlwapi.lib \
+        user32.lib uuid.lib winmm.lib winspool.lib, \
```

Comment:
same as above, I don't think sort oder is what is important here.

Reply:
It is not important for awt.dll. It is important in the context of all other calls to SetupJdkLibrary.